deploy.py

```diff
@@ -43,10 +46,11 @@ def run():
     )
 
     for region in regions:
-
+        print(region)
         lmb = boto3.client("lambda", region_name=region)
         cw_events = boto3.client("events", region_name=region)
         gd = boto3.client("guardduty", region_name=region)
+        print(gd.list_detectors()["DetectorIds"])
```

Comment:
More that should be only for testing. Maybe set up logger and set these to debug so we can get them when we want?